NitroxLauncher/Pages/ServerPage.xaml.cs

```diff
@@ -23,11 +24,30 @@ private void OnRequestNavigate(object sender, RequestNavigateEventArgs e)
             e.Handled = true;
         }
 
-        private void StartServer_Click(object sender, RoutedEventArgs e)
+        private void StartPublicServer_Click(object sender, RoutedEventArgs e)
         {
             try
             {
+                ServerConsolePage.isPrivateServer = false;
                 LauncherLogic.Instance.StartServer(RBIsExternal.IsChecked == true);
+                
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.ToString(), "Error", MessageBoxButton.OK, MessageBoxImage.Error);
```

Comment:
You should log the error to logs, to allow for future debugging. Although unlikely, the user might not remember the error when shown by the messagebox.